### PR TITLE
Fix and improve details on the setup and voting flow

### PR DESCRIPTION
**Pull request by andreslucena**
This PR introduces multiple changes mostly related to #35 and copies (from #32 and #33). 

Most notably: 

1. [View the number of credits available in Admin's steps](https://github.com/decidim-vocdoni/decidim-module-vocdoni/commit/77882c763ffde6d07bca09c0e89484033c45ea45) 
2. [Add "Fix it" links to SetupForm errors](https://github.com/decidim-vocdoni/decidim-module-vocdoni/commit/4af79883740f27c403f62bd97af840fcb7f4114a) - So it's easier to admins to find out where they can solve the error.
3. [Add Vocdoni logo on Admin's steps](https://github.com/decidim-vocdoni/decidim-module-vocdoni/commit/9e5d89154139ea4da0656da2fccab2cc43c39dec) and [Add Vocdoni logo on Voting booth](https://github.com/decidim-vocdoni/decidim-module-vocdoni/commit/1dd73fb1c6d1c2669b3dee50f9919778cf0e4c58)
4. [Add secretUntilTheEnd setting on Election](https://github.com/decidim-vocdoni/decidim-module-vocdoni/commit/95ca1e1e79036c1763dac22463614cd4495c8563) - This setting configures on the Vocdoni API if the election has to be
secret until the end date. If not, the results are visible in real-time.
5. [Change vote verification step on Voting booth](https://github.com/decidim-vocdoni/decidim-module-vocdoni/commit/ce73013483106e475bc2bf4ca9c6d18f1ffd7007)

There area also a couple refactors of the code to make it easier to maintain. 

As I said, there are lots of changes on little details, texts, and (mostly) mini-improvements on the flow. Most of them come from feedback given by  @ferranrego and @carolromero.

### CODE REVIEW

File: app/commands/decidim/vocdoni/admin/update_election.rb [hunk 1]
```diff
@@ -55,6 +55,18 @@ def update_election!
...
         end
+
+        def election_type_attributes
+          {
+            election_type: {
+              auto_start: form.auto_start,
+              interruptible: form.interruptible,
+              dynamic_census: form.dynamic_census,
+              secret_until_the_end: form.secret_until_the_end,
+              anonymous: form.anonymous
```
Comment (microstudi) [inline app/commands/decidim/vocdoni/admin/update_election.rb:66]: Some of these configurations are not in the frontend, I supose they will be in the future?

**Review by microstudi — APPROVED** [review]
Beautiful PR @andreslucena!
This really feels like a way to go.
I've been able to perform the whole process, I've encountered this strange thing having to add the url of the vocdoni api to be able to vote. Probably somthing strange with my dev app. Also I see the results in Decidim but looks that the vocdoni explorer does not show the results. 
Maybe we need to tell the api to publish the results?

**Comment by andreslucena** [comment]
> I've been able to perform the whole process

🥳 

> I've encountered this strange thing having to add the url of the vocdoni api to be able to vote. 

I understand that's related to the bug that we were talking last week where there's a first request that gives 400 when checking with the Census (but then it seems to work well after that)? Or is that you can't make **any** vote if you don't set up the URL? 

> Also I see the results in Decidim but looks that the vocdoni explorer does not show the results.

I've seen this kind of behavior from the Vote Explorer to. I think it's just a random bug on their side.

File: app/commands/decidim/vocdoni/admin/update_election.rb (hunk 1 above)
Comment (andreslucena) [reply app/commands/decidim/vocdoni/admin/update_election.rb:66]: Yes, at least `interruptible` for sure, as that's part of the grant and we need to support that. I'm working on this one on another PR.

See #35: 

> v. Manage votes on-demand according to the Election lifecycle states specifications as defined in [developer.vocdoni.io/get-started/intro#election-lifecycle-states](https://developer.vocdoni.io/get-started/intro#election-lifecycle-states)

The others are supported on the API, but at least some are not (yet) implemented in the SDK. I've got them from https://github.com/vocdoni/vocdoni-sdk#creating-a-voting-process

**Comment by andreslucena** [comment]
>  Maybe we need to tell the api to publish the results? 

At least with our current setup, we don't need to tell them to publish the results, as that's done automatically on their side when the "end time" has arrived.

**Comment by microstudi** [comment]
> I understand that's related to the bug that we were talking last week where there's a first request that gives 400 when checking with the Census (but then it seems to work well after that)? Or is that you can't make **any** vote if you don't set up the URL?

Yes, the same, the bug is that the object "client" does not sets up the property "url" according to the env variable.

File: app/commands/decidim/vocdoni/admin/update_election.rb (hunk 1 above)
Comment (microstudi) [reply app/commands/decidim/vocdoni/admin/update_election.rb:66]: great, let's leave it like this

**Pull request merged**